### DISCUSSION TITLE
cablage CE ACI

### DISCUSSION THREAD

**Discussion by ApiFelix** [FredM67/PVRouter-3-phase#85] [General]
bonjour
tout d'abord quelle est la différence entre carte ACI Kitable et non kitable ? comment les reconnait  on?

ensuite ...

je ne peux pas tirer de câble entre le routeur et le CE.
Par contre j'ai une seconde alimentation permanente au niveau du CE
Puis je 
- alimenter la carte ACI avec cet alim permanente
- câbler la résistance avec le câble en provenance du routeur ?

merci

**Comment by FredM67** [comment]
Il y a des photos [ici](https://github.com/FredM67/PVRouter-3-phase/wiki/Cartes-ACI-monophas%C3%A9e-et-triphas%C3%A9e
) ;-)
Oui, tu peux alimenter la carte avec une prise murale, mais il faudra alors rajouter un couple contacteur HC/disjoncteur 2A.
En aucun cas on ne doit by-passer le thermostat et donc la carte ACI.
Cela correspond au schéma [ici](https://github.com/FredM67/PVRouter-3-phase/blob/main/docs/CarteACI_Mono_Autre_Routeur.pdf)

**Reply by ApiFelix** [reply]
oups ... pas vu le wiki sorry .....
merci !

 un contacteur HC/disjoncteur2a entre la sortie du routeur et les broches blanches de l'ACI ?
pourquoi un contacteur HC alors que je veux le piloter par Solarcast ?

enfin (oui ça fait beaucoup de pourquoi ...) tu m'envoie [là](https://github.com/FredM67/PVRouter-3-phase/blob/main/docs/CarteACI_Mono_Autre_Routeur.pdf)
mais je suis en MK2pvRouteur. Alors plutôt [là](https://github.com/FredM67/PVRouter-3-phase/blob/main/docs/HowToInstall.pdf) non ?

**Reply by ApiFelix** [reply]
promis tu aura droit à une bière au Shluch ou au télégraphe !!!

**Reply by FredM67** [reply]
Il faut un contacteur **style** HC, mais un contacteur simple fait l'affaire aussi.
Comme on ne peut pas alimenter la carte ACI par du courant haché, et si on ne veut/peut pas faire un aller-retour de fils 2.5mm2 entre le CE et l'endroit où est installé le routeur, alors il faut découpler l'alim de la carte ACI de l'alim des résistances, tout en gardant la fonctionnalité thermostat ET sécurité thermique, c'est le rôle du contacteur.

Le cas "autre routeur" est en réalité le cas applicable de façon universelle mais obligatoire pour les routeurs qui alimentent le CE via leur propre alim.
Pour le mk2, on a le choix entre les schémas du HowToInstall (rajout d'un aller-retour de fils 2.5mm2) et les schémas "autre routeur".

**Reply by ApiFelix** [reply]
super !
j'ai (presque) tout compris
le disjoncteur tu le mets entre le contacteur et la carte ACI ou avant le contacteur ?

**Reply by FredM67** [reply]
Le disjoncteur 2A sert à protéger la bobine du contacteur.
J'ai mis à jour le premier schéma, je ferai les autres plus tard...
https://github.com/FredM67/PVRouter-3-phase/blob/main/docs/CarteACI_Mono.pdf

**Reply by ApiFelix** [reply]
👍👍👍👍👍

Merci !

**Reply by FredM67** [reply]
Tout compte fait, j'étais dans le moove, j'ai mis à jour les autres schémas !

**Reply by FredM67** [reply]
Bon, après avoir réfléchit, j'ai à nouveau changer le nom du fichier, car ce sont bien des schémas pour les autres routeurs.
Pour le mk2pvrouteur, c'est très légèrement différent.
Rien ne change par rapport au 2A du contacteur. J'essaie de faire les schémas aujourd'hui.

**Reply by FredM67** [reply]
J'ai aussi rajouté un avertissement. En effet, j'avais omis d'indiquer ce point, l'utilisation d'un contacteur en sortie de carte ACI présente une faille de sécurité. Certes, la probabilité est très faible, mais bon, elle existe !

**Reply by ApiFelix** [reply]
pour le MK2, je ne sais pas si c'est moi qui n'ai pas trouvé le doc ou si tu n'as pas encore terminé
merci en tous les cas

**Reply by FredM67** [reply]
Je ne vais pas faire de doc de suite pour le mk2 car il est vrai que cette histoire de faille, même si le risque est extrêmement faible, ne me satisfait pas, sachant que pour le mk2, une vraie solution sûre existe (rajout d'un aller-retour de fils 2.5mm2), ou alors config avec 2 boîtiers et câble type réseau entre les 2 boîtiers.

**Comment by FredM67** [comment]
> promis tu aura droit à une bière au Shluch ou au télégraphe !!!

toujours partant pour une bière et même plusieurs si je suis à pied ou à vélo :D

**Comment by ApiFelix** [comment]
comme je suis en MK2 je reviens sur ma question d'origine

> "Par contre j'ai une seconde alimentation permanente au niveau du CE
> Puis je
>     alimenter la carte ACI avec cet alim permanente
>     câbler la résistance avec le câble en provenance du routeur ?"

tu me réponds  :

> "Oui, tu peux alimenter la carte avec une prise murale, mais il faudra alors rajouter un couple contacteur HC/disjoncteur 2A."


comprends pas ...

- je veux virer le contacteur HC/HP de mon tableau et donc
- je déplace les fils qui alimentaient mon CE en partance du contacteur vers le disjoncteur 20A => mon CE est alimenté et chauffe ... en permanence
- j'utilise les câble de cette prise électrique (il n'y a rien d'autre sur cette ligne) comme lien entre le routeur et la platine de puissance : je déconnecte du disjoncteur pour les monter vers le routeur
maintenant, c'est le routeur qui pilote la chauffe

c'est mes yeux ou c'est marqué Gare d'Austerlitz ?

**Reply by ApiFelix** [reply]
et j'utilise le contacteur HC comme marche forcée manuelle en le connectant aux 2 cables du triac ?
(0 triac gère  1 triac shunté=marche forcée)

j'ai bon ?

**Comment by FredM67** [comment]
> comme je suis en MK2 je reviens sur ma question d'origine
> 
> > "Par contre j'ai une seconde alimentation permanente au niveau du CE
> > Puis je
> > alimenter la carte ACI avec cet alim permanente
> > câbler la résistance avec le câble en provenance du routeur ?"
> 
> tu me réponds :
> 
> > "Oui, tu peux alimenter la carte avec une prise murale, mais il faudra alors rajouter un couple contacteur HC/disjoncteur 2A."
> 
> comprends pas ...
> 
> * je veux virer le contacteur HC/HP de mon tableau et donc
Okay
> * je déplace les fils qui alimentaient mon CE en partance du contacteur vers le disjoncteur 20A => mon CE est alimenté et chauffe ... en permanence
Tout à fait !
> * j'utilise les câble de cette prise électrique (il n'y a rien d'autre sur cette ligne) comme lien entre le routeur et la platine de puissance : je déconnecte du disjoncteur pour les monter vers le routeur

Presque ! Les câbles de la prise électrique servent à alimenter la carte ACI en courant permanent non haché.

Les câbles de puissance qui arrivaient sur le bornier de la carte ACI doivent être débranchés et doivent maintenant passer à travers le contacteur.

À la place, tu branches sur le bornier de la carte ACI les câbles qui viennent de la prise.

Ensuite, parmi les câbles rouges et bleus qui sortent de la carte ACI, tu les coupes (ou les démontent de la résistance), tu regroupes les rouges coté ACI sur un WAGO, idem pour les bleus, tu fais la même chose côté résistance.

Des 2 wago côtés ACI, tu branches un rouge et un bleu sur un disjoncteur 2A, et tu ressorts de ce disjoncteur vers les bornes A1/A2 du contacteur que tu as récupéré de ton tableau.

Sur le contacteur, tu arrives avec les câbles de puissance qui étaient sur le bornier noir de la carte ACI et du ressort du contacteur vers les 2 Wago côté résistance (attention, section 2.5mm2 partout pour être sûr).
>   maintenant, c'est le routeur qui pilote la chauffe
> 
> c'est mes yeux ou c'est marqué Gare d'Austerlitz ?

**Reply by ApiFelix** [reply]
Mais dans une installation "pure mk2" je ne vois pas de contacteur entre le triac et les resistances

A disjoncteur alimente la carte ACi et le triac s'intercaler entre le bornier blanc et les resistances
Quelle différence ?

**Reply by FredM67** [reply]
Dans une installation avec mk2, normalement on fait un aller-retour de fils 2.5mm2 entre le chauffe-eau (carte ACI/résistance) et le boitier du routeur.
Ceux qui ne peuvent pas faire ca, ou ne veulent pas, choisissent en général une configuration en 2 boîtiers, un boitier carte-mère, et un boitier "sortie" placé juste à côté du chauffe-eau. De cette façon, pas besoin de tirer un aller-retour en 2.5mm2, par contre il faut relier les 2 boîtiers idéalement par un câble type réseau. On peut aussi faire en liaison radio, mais on perd un peu en réactivité.

**Reply by ApiFelix** [reply]
comme ça  :

![image](https://github.com/FredM67/PVRouter-3-phase/assets/60440843/b4f5a0b9-d348-4157-946d-48b48a912c2e)

**Reply by ApiFelix** [reply]
> Presque ! Les câbles de la prise électrique servent à alimenter la carte ACI en courant permanent non haché.

non
la carte ACI est alimentée par le disjoncteur ((en permanence vu que j'ai viré le contacteur HC)
 le câble de la prise remplace l'aller retour entre le routeur et les résistances

**Reply by FredM67** [reply]
Au tout début, tu disais que tu ne pouvais pas tirer de câble en plus, ET que tu avais une prise à proximité du CE.
Dans le cas, la seule solution, moyennant la faille indiquée, est de câbler comme pour les autres routeurs, enfin de façon similaire.

**Reply by ApiFelix** [reply]
effectivement je n'avais pas été très clair (j'ai déjà du mal à l’être dans ma tête ...)
oui cette prise était destinée a devenir le câble aller/retour du routeur

et du coups après tous ces échanges je me rend compte que je suis du coups dans un cas MK2 typique ...

merci pour ta patience !!

**Reply by FredM67** [reply]
Tu voulais dire que tu voulais "démonter" cette prise, et utiliser les fils qui sont entre cette prise ET le disjoncteur associé au tableau pour faire l'aller-retour ?!?
Alors, oui, mais niveau couleur, tu ne seras pas aux normes. Il faut 2 fils ni bleu, ni jaune/vert.
